File: perl/1.pl
#!/usr/bin/perl
use List::Util qw(sum);
use strict;

# Find the sum of all the multiples of 3 or 5 below 1000.

print sum grep { ($_ % 3 == 0) or ($_ % 5 == 0) } 1..999;
print "\n";

File: c/1.c
#include <stdio.h>

// Find the sum of all the multiples of 3 or 5 below 1000

int main(void){
	
	int i;
	int sum = 0;
	
	for(i = 1; i < 1000; i++){
		if(i % 3 == 0 || i % 5 == 0){
			sum = sum + i;
		}
	}
	
	printf("%d\n", sum);
	return 0;
}
